system-test/pubsub.ts

```diff
@@ -230,12 +230,11 @@ describe('pubsub', () => {
 
     it('should publish a message with attributes', async () => {
       const data = Buffer.from('raw message data');
-      const attrs = {
-        customAttribute: 'value',
-      };
-      const message = await publishPop(data, attrs);
-      assert.deepStrictEqual(message.data, data);
-      assert.deepStrictEqual(message.attributes, attrs);
+      const customAttribute = 'value';
+      const attrs = {customAttribute};
+      const m = await publishPop(data, attrs);
+      assert.deepStrictEqual(m.data, data);
+      assert.deepStrictEqual(m.attributes.customAttribute, customAttribute);
```

Comment:
I think just `assert.strictEqual` is good here since this is now a string===string test.

Reply:
I don’t think this change was made, unless GH is caching me!

Reply:
Doh, done!